### PR TITLE
feat: remove sv and cnv callers from annotation and filter rules

### PR DESCRIPTION
**Pull request by khurrammaqbool**
### This PR:

Removed: sv and cnv callers from annotation and filter

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

**Comment by khurrammaqbool** [comment]
Screenshot 2022-03-15 at 11 26 50
Screenshot 2022-03-15 at 11 28 23
Screenshot 2022-03-15 at 11 29 29
Screenshot 2022-03-15 at 11 30 14

File: CHANGELOG.rst [hunk 1]
```diff
@@ -31,6 +31,7 @@ Removed
 ^^^^^^^
 
 * ``--qc-metrics/--no-qc-metrics`` flag from the ``balsamic report deliver`` command #833
+* SV and CNVN callers from annotation and filtering #871
```
Comment (ivadym) [inline CHANGELOG.rst:34]:
```suggestion
* SV and CNV callers from annotation and filtering #871
```

**Review by ivadym — CHANGES_REQUESTED** [review]
👍 

Just a minor comment from my part: the deleted rules should be removed from the `DELIVERY_RULES` constant in `workflow_rules.py`

```
DELIVERY_RULES = [
     ... ,
    "bcftools_filter_manta",
    "bcftools_filter_delly",
    "bcftools_filter_ascat",
    "bcftools_filter_cnvkit",
    ... ,
]
```

**Comment by khurrammaqbool** [comment]
> 👍
> 
> Just a minor comment from my part:
> 
>     * The deleted rules should be removed from the `DELIVERY_RULES` constant in `workflow_rules.py`  and the `bcftools_filter_svdb` one included instead.
> 
> 
> ```
> DELIVERY_RULES = [
>      ... ,
>     "bcftools_filter_manta",
>     "bcftools_filter_delly",
>     "bcftools_filter_ascat",
>     "bcftools_filter_cnvkit",
>     ... ,
> ]
> ```

nice catch!

File: CHANGELOG.rst (hunk 1 above)
Comment (khurrammaqbool) [reply CHANGELOG.rst]: Thanks!

**Review by ivadym — APPROVED** [review]
👍 🥇

**Comment by ashwini06** [comment]
@khurrammaqbool : I was wondering if you had used the updated branch from develop for your branch `feat/remove_sv_vep`
Your panel outputs doesn't contain the newly added rule `vcf2cytosure_cnvkit` that was earlier merged with this [PR](https://github.com/Clinical-Genomics/BALSAMIC/pull/880)

**Pull request merged**